<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otshet_Dialog</class>
 <widget class="QDialog" name="Otshet_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Otshet_Dialog {
	background-color: rgb(183, 239, 237);
border: 3px solid #000000;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="20,7,7,7,7,7,7,7,21,10">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 3px solid #000000;
}

</string>
     </property>
     <property name="text">
      <string>    ЗАКАЗ № </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 1px solid #ffffff;
}

</string>
       </property>
       <property name="text">
        <string>    Заказчик: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 1px solid #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>    Судно №: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 1px solid #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>    Откуда:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 1px solid #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>    Куда:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 1px solid #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>    Дата отправления: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 1px solid #ffffff;
}
</string>
       </property>
       <property name="text">
        <string>    Время отправления: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
border: 0px solid #000000;
border-bottom: 1px solid #b7efed;
}

</string>
       </property>
       <property name="text">
        <string>    Груз:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #545454;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="70,30">
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
